common/test/acceptance/fixtures/discussion.py

```diff
@@ -170,3 +171,12 @@ def __init__(self, result):
 
     def get_config_data(self):
         return {"search_result": json.dumps(self.result)}
+
+
+class ForumsConfigMixin(object):
+    """Mixin providing a method used to configure the forums integration."""
+    def set_forums_configuration(self, is_enabled=True):
```

Comment:
This method reads a bit strangely when it's actually used. A couple of possible options to make it clearer:

```
def set_forums_configuration(self, **kwargs):
    ...

self.set_forums_configuration(enabled=True)
```
or
```
def enable_forums(self, is_enabled=True):
    ...

self.enable_forums()
```